sources/packages/backend/apps/api/src/services/student-scholastic-standings/student-scholastic-standings.service.ts

```diff
@@ -258,7 +262,7 @@ export class StudentScholasticStandingsService extends RecordDataModelService<St
       // Left as the last step to ensure that everything else was processed with
       // success and the notification will not be generated otherwise.
       if (createdRestriction) {
-        await this.studentRestrictionService.createNotifications(
+        await this.studentRestrictionsService.createNotifications(
```

Comment:
Most of the services seem to not be using the plural, why was it renamed to use the plural?

Reply:
yes, it was added due to the conflict with the same name class from API. it will be resolved when I implement the solution we discussed today `SharedService`.